diff --git a/src/lib/errors.js b/src/lib/errors.js
index 187d89c08b1b6e8a1c9c2f02128580f5fa4d52c7..45de197729dd1420c6e506f214f8af33ec623949 100644
--- a/src/lib/errors.js
+++ b/src/lib/errors.js
@@ -167,7 +167,7 @@ exports.LIT_ERROR = {
     NODE_ERROR: {
         name: 'NodeError',
         code: 'node_error',
-        kind: exports.LitErrorKind.Unknown,
+        kind: exports.LIT_ERROR_KIND.Unknown,
     },
     WALLET_SIGNATURE_NOT_FOUND_ERROR: {
         name: 'WalletSignatureNotFoundError',
@@ -202,17 +202,17 @@ exports.LIT_ERROR = {
     NETWORK_ERROR: {
         name: 'NetworkError',
         code: 'network_error',
-        kind: exports.LitErrorKind.Unexpected,
+        kind: exports.LIT_ERROR_KIND.Unexpected,
     },
     TRANSACTION_ERROR: {
         name: 'TransactionError',
         code: 'transaction_error',
-        kind: exports.LitErrorKind.Unexpected,
+        kind: exports.LIT_ERROR_KIND.Unexpected,
     },
     AUTOMATION_ERROR: {
         name: 'AutomationError',
         code: 'automation_error',
-        kind: exports.LitErrorKind.Unexpected,
+        kind: exports.LIT_ERROR_KIND.Unexpected,
     },
 };
 exports.LIT_ERROR_CODE = {
